src/remote.cc

```diff
@@ -143,6 +143,15 @@ class MsgWriter
         write(display_buffer.lines());
     }
 
+    inline void write_all() {}
+
+    template<typename Arg1, typename ...Args>
+    void write_all(Arg1&& arg1, Args&&... args)
+    {
+        write(std::forward<Arg1>(arg1));
+        write_all(std::forward<Args>(args)...);
+    }
+
```

Comment:
We have c++17 fold expressions !
```
template<typename... Args>
void write_all(Args&&... args)
{
    (write(std::forward<Args>(args)), ...);
}
```

Also, maybe we could make this method just called `write`, and make all the others private with another name (write_impl or so).

Reply:
Oh, that's neat.  I'm apparently still not yet caught up :D 

I made the internal methods `write_field` and `write_raw`.